package/rpm/SOURCES/usr/sth/conf/sth_default.conf

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Short Term History configuration variables.
+# STH is configured defining the folowing variables. 
+# All variables are optional, so, if one of this variables is not defined 
+# default value is used. 
+
+# See https://github.com/telefonicaid/IoT-STH#running-the-sth-server 
+# for further information
+
+# Note: is important to export environment variables with export
+
+# The host where the STH server will be started. Default value: "localhost".
+export STH_HOST="0.0.0.0"
+
+# The port where the STH server will be listening. Default value: 8666.
+export STH_PORT="8666"
+
+# The logging level of the messages. Messages with a level equal or superior 
+# to this will be logged. Allowed values are: .Default value: "info".
+export LOG_LEVEL="info"
+
+# A flag indicating if the logs should be sent to the console. Default value: true.
+export LOG_TO_CONSOLE="false"
+
+# A flag indicating if the logs should be sent to a file. Default value: true.
+export LOG_TO_FILE="true"
+
+# Maximum size in bytes of the log files. If the maximum size is reached, a new log file is created incrementing a counter used as the suffix for the log file name. Default value: undefined.
+# 20MB=20*1024*1024=20971520
+export LOG_FILE_MAX_SIZE_IN_BYTES="20971520"
+
+# The path to a directory where the log file will be searched for or created if it does not exist. Default value: "./log"
+export LOG_DIR="/var/log/sth"
+
+# The name of the file where the logs will be stored. Default value: "sth_app.log".
+export LOG_FILE_NAME="sth_default.log"
+
+# The prefix to be added to the service for the creation of the databases. More information below. Default value: "sth".
+export DB_PREFIX="sth"
+
+# The service to be used if not sent by the Orion Context Broker in the notifications. Default value: "orion".
+export SERVICE="orion"
```

Comment:
I've seen this was fixed in https://github.com/telefonicaid/IoT-STH/commit/1b710efe40b04c6944ce7a18de42fa3ebcc8807d